apps/vscode/editor/src/app.tsx

```diff
@@ -4,14 +4,44 @@ import { Tldraw, TldrawApp, TDFile, TDDocument } from '@tldraw/tldraw'
 import { vscode } from './utils/vscode'
 import { defaultDocument } from './utils/defaultDocument'
 import type { MessageFromExtension, MessageFromWebview } from './types'
+import {toSVG, fromSVG} from "./utils/svgEmbedder";
```

Comment:
linter needed here,
```suggestion
import { toSVG, fromSVG } from './utils/svgEmbedder';
```

Reply:
I think our auto linter isn't running on push. Thanks for the eyes though!